src/CommonDBTM.php

```diff
@@ -5955,7 +5955,7 @@ public function getMassiveActionsForItem(): MassiveAction
             $params['entity_restrict'] = $this->getEntityID();
         }
 
-        return new MassiveAction($params, [], 'initial', true);
+        return new MassiveAction($params, [], 'initial', $this->fields['id']);
```

Comment:
```suggestion
        return new MassiveAction($params, [], 'initial', ($this->fields['id'] > 0));
```

we're waiting for a boolean, maybe let the parameter clear, no ?

Reply:
No, an `id` has to be used to be able to filter single actions, see `MassiveAction::getAllMassiveActions()`.
I will fix PHPDoc and expected types to make this clear.

Reply:
ok

Reply:
See last commit. I found another bad param usage in API, so I guess changing the signature is usefull.